common/views/components/IIIFViewer/ViewerSidebar.tsx

```diff
@@ -56,6 +56,7 @@ const AccordionInner = styled(Space).attrs({
     &:active,
     &:focus {
       outline: 0;
+      box-shadow: ${props => props.theme.focusBoxShadow};
```

Comment:
Is it worth boxing this style into `focussable` or similar asit's being [used in a few places](https://github.com/wellcomecollection/wellcomecollection.org/search?q=focusBoxShadow)?